ddp-workspace/projects/ddp-osteo/src/main.ts

```diff
@@ -4,8 +4,6 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
-import 'hammerjs';
```

Comment:
Got a warning here:
```
WARNING in C:\epam\ddp-angular\ddp-workspace\projects\ddp-osteo\src\main.ts depends on 'hammerjs'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```
Instead of `import 'hammerjs';` added `HammerModule` in the osteo `app.module.ts `